debian/rules

```diff
@@ -32,6 +36,7 @@ override_dh_shlibdeps:
 
 override_dh_virtualenv:
 	dh_virtualenv $(DH_VENV_ARGS)
+	debian/matrix-synapse-py3/opt/venvs/matrix-synapse/bin/python -B -m twisted.trial --reporter=text -j 2 ../tests/
```

Comment:
because our current working dir is the `debian/` directory

Reply:
I don't think it is?

Reply:
(if it is, why is the command `*debian*/matrix-synapse-py3/opt/venvs/matrix-synapse/bin/python`?)

Reply:
ok, the reason this works is because [twisted.python.reflect.filenameToModuleName](https://github.com/twisted/twisted/blob/trunk/src/twisted/python/reflect.py#L325) falls back to returning the final component of the path (`tests` in this case) if it can't find a matching python package anywhere else.